src/types/index.ts

```diff
@@ -531,6 +535,11 @@ export interface IKeyPair {
   secretKey: string;
 }
 
+export interface Label {
```

Comment:
In my opinion the name of this interface is too short, not enough informative and lacks the `I` prefix.